translations/de/pack/core2.de.json

```diff
@@ -0,0 +1,397 @@
+[
```

Comment:
This should really be in a separate PR. At the very least, it should not be repeated in 5 PRs.